server/public/layout.css

```diff
@@ -84,6 +94,16 @@ form {
   form {
     width: 80%;
   }
+
+  .fa-solid {
+    color: #6610f2;
+    font-size: 18px;
+  }
+
+  .fa-regular {
+    font-size: 18px;
+    color: #6610f2;
+  }
```

Comment:
In general you should avoid augmenting classes from external libraries and instead add your own.